eth-wallpaper-gen/main.py

```diff
@@ -0,0 +1,70 @@
+import sys
+from PIL import Image, ImageDraw
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+def scale(ms, width, height):
+    return [(x*width, y*height) for (x, y) in ms]
+
+magics = {
+    "top-left":     [(0.498, 0.183), (0.393, 0.445), (0.498, 0.538), (0.498, 0.183)],
+    "top-right":    [(0.498, 0.183), (0.604, 0.445), (0.498, 0.538), (0.498, 0.183)],
+    "bottom-left":  [(0.392, 0.472), (0.498, 0.567), (0.498, 0.697), (0.392, 0.472)],
+    "bottom-right": [(0.605, 0.472), (0.498, 0.567), (0.498, 0.697), (0.605, 0.472)],
+}
+
+mirrors = {
+    "top-left":     "bottom-right",
+    "top-right":    "bottom-left",
+    "bottom-left":  "top-right",
+    "bottom-right": "top-left",
+}
+
+im = Image.open(sys.argv[1])
```

Comment:
We probably want to organize all of this logic into methods and implement a `main` method that calls the specific functions.

Reply:
Done :+1: